resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -611,15 +611,16 @@ sam.init.execution_error=Could not execute `sam init`!
 sam.init.generating.template=Generating SAM template...
 sam.init.name=AWS Serverless Application
 sam.init.node_interpreter.label=Node interpreter:
-sam.init.project_sdk.label=Project SDK:
+sam.init.sdk.label=SDK:
+sam.init.sdk.error=Check SDK settings
 sam.init.python.bad_sdk=A valid Python SDK was not selected
 sam.init.sam_template.tooltip=The name of the AWS Serverless Application Model (AWS SAM) template to use.
 sam.init.schema.aws_credentials.label=AWS Connection:
 sam.init.schema.aws_credentials_missing=AWS credentials required to choose a Schema. Click here to pick profile.
 sam.init.schema.aws_credentials_select=Select AWS Credentials
 sam.init.schema.aws_credentials_select_region=Select AWS Region
 sam.init.schema.label=Event Schema:
-sam.init.schema.pleaseSelect=Choose schema of the EventBridge serverless event...
+sam.init.schema.pleaseSelect=Choose the schema of the EventBridge serverless event
```

Comment:
`Choose the EventBridge serverless event schema`?

Reply:
Changed